getting_started/step_by_step/scripting_player_input.rst

```diff
@@ -116,6 +138,39 @@ Here is the complete ``Sprite2D.gd`` file for reference.
 
         position += velocity * delta
 
+ .. code-tab:: csharp C#
+
+    using Godot;
+    
+    public class Sprite : Godot.Sprite
```

Comment:
The class should be `Sprite2D` since documentation targets master first, for cherry-picking it would have to be replaced by `Sprite`.

Reply:
According to the description of the `Sprite` class, it says a node that displays a 2D texture. I think the name omits `2D`. 
Otherwise, VS Code can't find a reference, and the project is not building when I use the `Sprite2D` class.

Can you please correct/advise?

Reply:
You must be using Godot 3.x, in 4.0 the class has been renamed to `Sprite2D`. The documentation targets `master` so it should use the 4.0 name.

Reply:
done